src/App.tsx

```diff
@@ -1,9 +1,15 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(
+    movie => movie.title.toLowerCase().includes(query)
```

Comment:
de-structuring would save few chars
```suggestion
    { title, description } => title.toLowerCase().includes(query)
```